crates/spirv-attrib/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+#![cfg_attr(target_arch = "spirv", feature(register_attr), register_attr(spirv))]
```

Comment:
I don't think this does anything?

Reply:
Why not? It only registers the spriv attribute on spirv architecture.

Reply:
Oh your right 👍